content/en/projects/trier-help.md

```diff
@@ -0,0 +1,22 @@
+---
+author: 'Hugo Authors'
+title: 'Help in Trier'
+thumbnail: '/projectsImg/10.jpg'
+---
+
+Hugo ships with several [Built-in Shortcodes](https://gohugo.io/content-management/shortcodes/#use-hugos-built-in-shortcodes) for rich content, along with a [Privacy Config](https://gohugo.io/about/hugo-and-gdpr/) and a set of Simple Shortcodes that enable static and no-JS versions of various social media embeds.
```

Comment:
Let's delete all not relevant texts from pages.
It would be easier to see what's empty.